### PR TITLE
[PORT] Telecomms Specialist

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

This is my first PR, so I fully expect there to be things I can optimize or fix, just let me know!
Adds a new lightweight role to engineering, the Telecomms Specialist! Monitor and repair telecomms, the tram, and other wireless devices when needed, watch out for suspicious activity on the network.

Does not alter any existing roles or content and simply add icons and code for this job in specific.

## How This Contributes To The Skyrat Roleplay Experience

Other departments have more laid-back roles such as the Coroner/Virologist, Barber, and even Corrections Officer. Telecomms is a somewhat underutilized mechanic in general outside of simply managing whether or not it works. By staffing the telecomms control room, the specialist has access to PDA messages, and the logged conversations on every station-side channel. While not useful most of the time, I believe there is room for emergent roleplaying with this role.

The expectations of the specialist are to: Configure, repair, and otherwise maintain critical station telecomms equipment, including setting up new relays. They are additionally the point of contact for security looking for evidence via PDA and comms messages to prove criminal intent. Also as a side bonus, the telecomms specialist is responsible for the tram controls system on tram station (it's telecomms access).

The telecomms specialist begins the round with a reduced engineering kit: only a screwdriver, multitool, wirecutters, and a mod-link scryer. They also get the monitor decryption key for the message monitor console. The specialist has reduced access to engineering, such as no access to APCs, air alarms, or engineering gear. During skeleton crew shifts, their access is increased to a full engineer.

Alternative titles include: "Wireless Operator, Network Engineer, Sysadmin, and Tram Technician"

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

image

image

image

image

image

</details>

## Changelog

:cl: SciDragon
add: Telecomms Specialist job, with a niche carved out of an underutilized engineering department.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm [hunk 1]
```diff
@@ -20,6 +20,23 @@
 	armor_type = /datum/armor/jacket_engi
 	allowed = list(/obj/item/flashlight, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/t_scanner, /obj/item/construction/rcd, /obj/item/pipe_dispenser, /obj/item/toy, /obj/item/storage/fancy/cigarettes, /obj/item/lighter)
 
+/obj/item/clothing/suit/toggle/jacket/tcomm
+	name = "telecomms jacket"
+	desc = "A comfortable jacket in engineering yellow with blue telecomms trim."
+	icon_state = "tcomm_dep_jacket"
+	armor_type = /datum/armor/jacket_engi
+	allowed = list(
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm:28]: Does this let nitrogen tank be held aswell?

File: modular_skyrat/modules/telecomms_specialist/telecomms_specialist.dm [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	liver_traits = list(TRAIT_ENGINEER_METABOLISM)
+
+	display_order = JOB_DISPLAY_ORDER_TELECOMMS_SPECIALIST
+	bounty_types = CIV_JOB_ENG
+	departments_list = list(
+		/datum/job_department/engineering,
+		)
+
+	family_heirlooms = list(
+		/obj/item/modular_computer/laptop,
```
Comment (projectkepler-ru) [inline modular_skyrat/modules/telecomms_specialist/telecomms_specialist.dm:30]: Could add a multitool

Comment (lessthnthree) [reply modular_skyrat/modules/telecomms_specialist/telecomms_specialist.dm:30]: Ye

File: modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm (hunk 1 above)
Comment (lessthnthree) [reply modular_skyrat/modules/customization/modules/clothing/under/utility_port/suits_port.dm:28]: nope. what are vox.

**Review by SpaceLoveSs13 — APPROVED** [review]

**Pull request merged**